nativescript/webpack.config.js

```diff
@@ -0,0 +1,64 @@
+var bundler = require("nativescript-dev-webpack");
+var CopyWebpackPlugin = require('copy-webpack-plugin');
+var path = require('path');
+
+module.exports = bundler.getConfig({
+  resolveloader: {
+    root: path.join(__dirname, "node_modules")
+  },
+  plugins: [
+    new CopyWebpackPlugin([
+      {
+        from: {
+          glob: 'assets/**/*',
+          dot: true
+        }
+      },
+      {
+        from: {
+          glob: 'css/**/*',
+          dot: true
+        }
+      },
+      {
+        from: {
+          glob: 'fonts/**/*',
+          dot: true
+        }
+      },
+      {
+        from: 'app.css',
+      },
+      {
+        from: 'app.css.' + process.env.PLATFORM,
+        to: 'app.css',
+      },
+      {
+        from: {
+          glob: '**/*.component.css',
+          dot: true
+        }
+      },
+      {
+        from: {
+          glob: '**/*.component.html',
+          dot: true
+        }
+      },
+      {
+        from: {
+          glob: '**/*.component.*.' + process.env.PLATFORM,
+          dot: true
+        },
+        to: '[path]/[name]'
+      },
+      // @nota plugins
```

Comment:
Wupsy. this shouldn't have been added to the seed project. :-1: 
This is one of the modules, we use in LYT3, the project I'm working on.